internal/view/details.go

```diff
@@ -294,9 +294,10 @@ func (d *Details) saveCmd(evt *tcell.EventKey) *tcell.EventKey {
 
 func (d *Details) cpCmd(evt *tcell.EventKey) *tcell.EventKey {
 	d.app.Flash().Info("Content copied to clipboard...")
-	if err := clipboard.WriteAll(d.text.GetText(true)); err != nil {
+	if err := clipboard.Init(); err != nil {
 		d.app.Flash().Err(err)
 	}
+	clipboard.Write(clipboard.FmtText, []byte(d.text.GetText(true)))
```

Comment:
In case of an error, we should write the error somewhere. Is it possible?

Reply:
Write function of this new pkg does not return error. 👀 
https://pkg.go.dev/golang.design/x/clipboard#Write

Reply:
It returns nil if error and chanel otherwise